client/src/pages/Membership/HowToJoin/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Typography,
+  createStyles,
+  WithStyles,
+  withStyles,
+  Box,
+  Button,
+} from '@material-ui/core';
+import Step from './Step';
+import ProfileSrc from './Profile.png';
+import MoneySrc from './Money.png';
+import BenefitsSrc from './Benefits.png';
+
+const styles = () =>
+  createStyles({
+    title: {
+      fontWeight: 'bold',
+    },
+  });
+
+type Props = WithStyles<typeof styles>;
+
+const HowToJoin: React.FC<Props> = ({ classes }: Props) => {
+  return (
+    <div>
+      <Typography variant="h2" className={classes.title} align="center">
+        How Do I Join ACS?
+      </Typography>
+      <Box display="flex" justifyContent="space-around" flexWrap="wrap">
+        <Step
+          stepNumber={1}
+          text="Enter Your Information"
+          icon={ProfileSrc}
+          iconAlt="Profile"
+        />
+        <Step
+          stepNumber={2}
+          text="Pay Membership Fees Later"
+          icon={MoneySrc}
+          iconAlt="Money"
+        />
+        <Step
+          stepNumber={3}
+          text="Enjoy All Of The Benefits"
+          icon={BenefitsSrc}
+          iconAlt="Benefits"
+        />
+      </Box>
+      <Box marginY={12} display="flex" justifyContent="center">
+        <Button component={Link} to="/membership/signup" variant="outlined">
```

Comment:
Yeah we definitely should but I think the way our `BWButton` is set up, for some reason the component prop isn't working. I think it's a weird thing with `withStyles` not really sure https://github.com/mui-org/material-ui/issues/16887

I'm thinking maybe a better solution would be to override the styles for the base `MuiButton` in the theme. Maybe?

Reply:
Hmm. 2 questions
1. Would we ever want a `Button` that is not a `BWButton`? If not then sure we can make the change. Otherwise we should keep the set up we have now.
2. Is it necessary to have `component={Link} to="/membership/signup"` rather than `href="/membership/signup"` as the prop? Is there a performance advantage of using `Link`?

Reply:
Ahh maybe not with react-router. With other routing solutions like in NextJS it's used to preload the pages the rendered `Link`s link to but I don't think react-router does any of that cool stuff. It would just give us access to their `Link` api.

Reply:
for 1

I dont think so 😅